source/jormungandr/tests/direct_path_asgard_integration_tests.py

```diff
@@ -201,43 +207,103 @@ def bss_route_response(request):
     return response
 
 
+def alternative_route_response(profiles_params):
+    response = response_pb2.Response()
+    response.response_type = response_pb2.ITINERARY_FOUND
+    for params in profiles_params:
+        journey = response.journeys.add()
+        journey.tags.append(params.profile_tag)
+        journey_response(journey, params.origin_mode)
+    return response
+
+
 def valid_response(request):
     if request.requested_api == type_pb2.direct_path:
-        if request.direct_path.streetnetwork_params.origin_mode == "bss":
+        if request.direct_path.profiles_params[0].origin_mode == "bss":
             return bss_route_response(request)
-        return route_response(request.direct_path.streetnetwork_params.origin_mode)
+        if len(request.direct_path.profiles_params) == 1:
+            return route_response(request.direct_path.profiles_params[0].origin_mode)
+        if len(request.direct_path.profiles_params) == len(DIRECT_PATH_ALTERNATIVES_PROFILES):
+            return alternative_route_response(request.direct_path.profiles_params)
```

Comment:
Before, it was always returning `route_response()` but now, it will only return when there is `1` or `len(DIRECT_PATH_ALTERNATIVES_PROFILES)`. What about cases when it's other values?

Reply:
`None` will be returned